sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/EchoAction.java

```diff
@@ -37,4 +40,16 @@ public Effect<Message> stringMessage(@PathVariable String msg) {
     String response = this.parrot.repeat(msg);
     return effects().reply(new Message(response));
   }
+
+  @GetMapping("/echo/repeat/{msg}/times/{times}")
+  public Flux<Effect<Message>> stringMessage(
+      @PathVariable String msg, @PathVariable Integer times) {
+
+    Stream<Effect<Message>> messages =
+        IntStream.rangeClosed(1, times)
+            .mapToObj(i -> new Message(parrot.repeat(msg)))
+            .map(m -> effects().reply(m));
+
+    return Flux.fromStream(messages);
```

Comment:
I think Flux.range(1. times).flatmap... would be better

Reply:
Yep, just discovered this method and also forced an async boundary.